### PR TITLE
{bio}[intel/2018b] UMI-tools v1.0.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2444.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3b742f6be6da6869b7226af2a76f4989 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2041.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/21907c0c35db8840b5e9808c9c6bce54 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2444.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2af12a13cf164ed728e5b459406aff90 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3131.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/faafcb5ebfb51c2f8d168a948ba4a856 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2041.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6f71afe8f8b5ff71e27a405dd0a23f9e for a full test report.

File: easybuild/easyconfigs/u/UMI-tools/UMI-tools-1.0.0-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('matplotlib', '3.0.0', versionsuffix),
+    ('Pysam', '0.15.1', versionsuffix),
+]
+
+#use_pip = True
```
Comment (vanzod) [inline easybuild/easyconfigs/u/UMI-tools/UMI-tools-1.0.0-intel-2018b-Python-3.6.6.eb]: Is there a specific reason we cannot use `pip` here?

Comment (boegel) [reply easybuild/easyconfigs/u/UMI-tools/UMI-tools-1.0.0-intel-2018b-Python-3.6.6.eb:18]: I guess it didn't work at the time, for some reason, don't recall the details.

**Comment by boegel** [comment]
Deprecated toolchain + we have easyconfigs for newer versions of UMI-tools already (cfr. #10825), so closing this

**Pull request closed without merging**